main.go

```diff
@@ -28,21 +34,33 @@ func main() {
 	headers := make([]wire.BlockHeader, len(headerStrings))
 
 	for id, headerStr := range headerStrings {
-		h, _ := BlockHeaderFromHex(headerStr)
+		h, _ := btclightclient.BlockHeaderFromHex(headerStr)
 		headers[id] = h
 	}
 
-	headerInsert, _ := BlockHeaderFromHex("01000000ea0ec14effa5f7f2a1a9f4431588b63b575d167a261c1d93b604000000000000c1844859aa7bb44251cf04a19098169f657e4bd91ebeb3f2a028211f1f8bde271c6e8250ef75051a7dc08785")
+	headerInsert, _ := btclightclient.BlockHeaderFromHex("01000000ea0ec14effa5f7f2a1a9f4431588b63b575d167a261c1d93b604000000000000c1844859aa7bb44251cf04a19098169f657e4bd91ebeb3f2a028211f1f8bde271c6e8250ef75051a7dc08785")
 
-	btcLC := NewBTCLightClientWithData(&chaincfg.MainNetParams, headers, startHeight)
+	btcLC := btclightclient.NewBTCLightClientWithData(&chaincfg.MainNetParams, headers, startHeight)
 	btcLC.Status()
 
+	rpcService := rpcserver.NewRPCServer(btcLC)
+	log.Info().Msgf("RPC server running at: %s", rpcService.URL)
+
 	if err := btcLC.InsertHeader(headerInsert); err != nil {
-		fmt.Println(err)
+		log.Err(err).Msg("Failed to insert block header")
 	} else {
-		fmt.Println("Insert success")
+		log.Info().Msgf("Inserted block header %s", headerInsert.BlockHash())
 	}
 
-	btcLC.CleanUpFork()
+	err := btcLC.CleanUpFork()
+	if err != nil {
+		log.Err(err).Msg("Failed to clean up fork")
+	}
 	btcLC.Status()
```

Comment:
It's better to remove this one. I used the `main` function for the test proposal, so the code here is unnecessary if you run PRC code here.